website/content/docs/connect/cluster-peering/usage/create-cluster-peering.mdx

```diff
@@ -0,0 +1,834 @@
+
+<Tabs>
+<Tab heading="HTTP API" group="api">
+
+1. In `cluster-01`, use the [`/peering/token` endpoint](/api-docs/peering#generate-a-peering-token) to issue a request for a peering token.
+
+  ```shell-session
+  $ curl --request POST --data '{"Peer":"cluster-02"}' --url http://localhost:8500/v1/peering/token
+  ```
+
+  The CLI outputs the peering token, which is a base64-encoded string containing the token details.
+
+1. Create a JSON file that contains the first cluster's name and the peering token.
+
+  <CodeBlockConfig filename="peering_token.json" hideClipboard>
+
+  ```json
+  {
+    "Peer": "cluster-01",
+    "PeeringToken": "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhZG1pbiIsImF1ZCI6IlNvbHIifQ.5T7L_L1MPfQ_5FjKGa1fTPqrzwK4bNSM812nW6oyjb8"
+  }
+  ```
+
+  </CodeBlockConfig>
+
+</Tab>
+
+<Tab heading="Consul CLI" group="cli">
+
+1. In `cluster-01`, use the [`consul peering generate-token` command](/commands/peering/generate-token) to issue a request for a peering token.
+
+  ```shell-session
+  $ consul peering generate-token -name cluster-02
+  ```
+
+  The CLI outputs the peering token, which is a base64-encoded string containing the token details.
+
+1. Save this value to a file or clipboard to use in the next step on `cluster-02`.
+
+</Tab>
+
+<Tab heading="Consul UI" group="ui">
+
+To begin the cluster peering process, generate a peering token in one of your clusters. The other cluster uses this token to establish the peering connection.
+
+Every time you generate a peering token, a single-use secret for establishing the secret is embedded in the token. Because regenerating a peering token invalidates the previously generated secret, you must use the most recently created token to establish peering connections.
+
+1. In the Consul UI for the datacenter associated with `cluster-01`, click **Peers**.
+1. Click **Add peer connection**.
+1. In the **Generate token** tab, enter `cluster-02` in the **Name of peer** field.
+1. Click the **Generate token** button.
+1. Copy the token before you proceed. You cannot view it again after leaving this screen. If you lose your token, you must generate a new one.
+
+</Tab>
+
+<Tab heading="Kubernetes" grou="k8s">
+
+1. In `cluster-01`, create the `PeeringAcceptor` custom resource. To ensure cluster peering connections are secure, the `metadata.name` field cannot be duplicated. Refer to the peer by a specific name.
+
+  <CodeBlockConfig filename="acceptor.yaml">
+
+  ```yaml
+    apiVersion: consul.hashicorp.com/v1alpha1
+    kind: PeeringAcceptor
+    metadata:
+      name: cluster-02 ## The name of the peer you want to connect to
+    spec:
+      peer:
+        secret:
+          name: "peering-token"
+          key: "data"
+          backend: "kubernetes"
```

Comment:
can you remove the two extra spaces before each line? it looks slightly off on the render

<img width="674" alt="image" src="https://user-images.githubusercontent.com/7556325/220277344-f35b58da-8eca-438b-be0c-b66cde04a10b.png">


same for the other k8s file